### PR TITLE
always show role icon with role

### PR DESCRIPTION
**Pull request by birdonfire**
minor change to game results message too

### CODE REVIEW

**Comment by ngenator** [comment]
Nice, I was thinking this stuff should be consolidated

File: lib/werewolf/slackbot.rb [hunk 1]
```diff
@@ -3,6 +3,24 @@
 module Werewolf
   class SlackBot < SlackRubyBot::Server
     attr_writer :channel
+    cattr_reader :role_icons
```
Comment (dissemble) [inline lib/werewolf/slackbot.rb]: you don't use this anywhere?   is this just for testing?

File: lib/werewolf/slackbot.rb [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+
+    @@role_icons = {
```
Comment (dissemble) [inline lib/werewolf/slackbot.rb]: this is, in fact, a constant.  right?  you wouldn't want someone to change this at runtime?

File: lib/werewolf/slackbot.rb (hunk 1 above)
Comment (birdonfire) [inline lib/werewolf/slackbot.rb]: fixt

**Pull request merged**